tgstation.dme

```diff
@@ -3734,6 +3734,16 @@
 #include "fulp_modules\main_features\mentorhelp\mentorpm.dm"
 #include "fulp_modules\main_features\mentorhelp\mentorsay.dm"
 #include "fulp_modules\main_features\mentorhelp\mentorwho.dm"
+#include "fulp_modules\main_features\species\species_integration.dm"
```

Comment:
You included all the new files in the dme but didnt remove all the old ones that were deleted during relocation. Is this intended for the sake of testing?

Reply:
yeah its just for testing atm